src/packages/core/data-type/components/data-type-input/data-type-input.element.ts

```diff
@@ -62,7 +62,7 @@ export class UmbDataTypeInputElement extends FormControlMixin(UmbLitElement) {
 	@property()
 	public set value(idsString: string) {
 		// Its with full purpose we don't call super.value, as thats being handled by the observation of the context selection.
-		this.selectedIds = idsString.split(/[ ,]+/);
+		this.selectedIds = idsString !== '' ? idsString.split(/[ ,]+/) : [];
```

Comment:
I see how this removes whitespace from IDs. But if someone came with this: " 23, , 123" you would still get an empty string entry in the middle of the array. I'm not sure if you like to solve that case as well?

Reply:
A utility function has been added: splitStringToArray and tests has been added